inc/mlx90632.h

```diff
@@ -161,6 +164,18 @@
 #define MLX90632_MTYP_MEDICAL 0
 #define MLX90632_MTYP_EXTENDED 17
 
+/* Refresh rates */
+typedef enum MLX90632_MEAS {
```

Comment:
Although not important, but just to keep it aligned with rest make it lowercase with underscore as separator and  add `_e` postfix for typdef to be a bit more POSIX alike...

```suggestion
typedef enum mlx90632_meas_e {
```